.gitignore

```diff
@@ -0,0 +1,3 @@
+.vscode
+.editorconfig
+.bzr
```

Comment:
I am just curious: Why do you have that file or directory in a git repository?

Reply:
We don't. The objective is to ensure we don't add it by mistake

Reply:
Understood, thank you!

FWIW, I do not think we should list files just because they should not be added to the repository. The list of such files is nearly infinite. Fortunately, policies suggested so far did not include such files IIRC.

Sorry for using words. I will try the [suggested](https://github.com/squid-cache/squid/pull/659#issuecomment-640605278) "iterate in code" approach from now on (for this PR) -- I noticed that suggestion after writing the above.